cmd/humioctl/filter_alerts_list.go

```diff
@@ -0,0 +1,63 @@
+// Copyright © 2020 Humio Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"strings"
+
+	"github.com/humio/cli/cmd/internal/format"
+	"github.com/spf13/cobra"
+)
+
+func newFilterAlertsListCmd() *cobra.Command {
+	cmd := cobra.Command{
+		Use:   "list [flags] <view>",
+		Short: "List all filter alerts in a view.",
+		Args:  cobra.ExactArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
+			view := args[0]
+			client := NewApiClient(cmd)
+
+			filterAlerts, err := client.FilterAlerts().List(view)
+			exitOnError(cmd, err, "Error fetching filter alerts")
+
+			actions, err := client.Actions().List(view)
```

Comment:
Consider if it should be possible to list filter alerts, if the user cannot list actions?

Reply:
Those require two different permissions.

Reply:
It is possible to retrieve the action names as part of the GraphQL query that finds the filter alert, so I think that we should do so. The user might not have permissions to list actions.

Reply:
The same is true for normal alerts and scheduled searches.